packages/vuetify/src/components/VSwitch/__tests__/VSwitch.spec.ts

```diff
@@ -1,93 +1,112 @@
-import { test, touch } from '@/test'
-import VSwitch from '@/components/VSwitch'
+// Components
+import VSwitch from '../VSwitch'
+
+// Utilities
+import {
+  mount,
+  Wrapper
+} from '@vue/test-utils'
+import { touch } from '../../../../test'
+
+// Types
+import { ExtractVue } from '../../../util/mixins'
+
+describe('VSwitch.js', () => {
+  type Instance = ExtractVue<typeof VSwitch>
+  let mountFunction: (options?: object) => Wrapper<Instance>
+
+  beforeEach(() => {
+    mountFunction = (options = {}) => {
+      return mount(VSwitch, {
+        ...options
+      })
+    }
+  })
 
-test('VSwitch.js', ({ mount }) => {
   it('should set ripple data attribute based on ripple prop state', async () => {
-    const wrapper = mount(VSwitch, {
+    const wrapper = mountFunction({
       propsData: {
         inputValue: false,
         ripple: false
       }
     })
 
-    let ripple = wrapper.find('.v-input--selection-controls__ripple')
+    let ripple = wrapper.findAll('.v-input--selection-controls__ripple')
 
-    expect(ripple.length).toBe(0)
+    expect(ripple).toHaveLength(0)
 
     wrapper.setProps({ ripple: true })
 
-    ripple = wrapper.first('.v-input--selection-controls__ripple')
-
-    await wrapper.vm.$nextTick()
```

Comment:
if you add back in `await wrapper.vm.$nextTick()` on line 39, your tests will pass

Reply:
Great! Thanks!